community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/commands/expressions/Pow.scala

```diff
@@ -20,7 +20,7 @@
 package org.neo4j.cypher.internal.compiler.v3_0.commands.expressions
 
 case class Pow(a: Expression, b: Expression) extends Arithmetics(a, b) {
```

Comment:
I wonder why Pow is in a separate file that MathFunction.scala? We could use NullAndNaNSafeMathFunction here? Or fix this in Arithmetics?